DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -15,6 +17,12 @@ public interface IRegistrationService
             string baseUrl,
             string userIp
         );
+
+        string RegisterAdminDelegate(
+            RegistrationModel registrationModel
+        );
+
+        bool RegisterCentreManager(RegistrationModel registrationModel);

```

Comment:
I'd probably call this TryRegisterCentreManager since it can fail and return false. (Like a TryParse does)